### ISSUE TITLE
Monitor pyth and track uptime

### ISSUE THREAD

**Issue by brittcyr** [Dual-Finance/status#28]
It is a hard dependency for us, so we should make sure that we are able to handle when we do not have price feeds
But we should also have monitoring and alerting so we can escalate when there are issues

**Comment by brittcyr** [comment]
They provide
https://github.com/pyth-network/pyth-observer
to build monitoring

**Comment by DonDuala** [comment]
Chainlink & Switchboard also now need to be tracked

**Comment by brittcyr** [comment]
> Chainlink & Switchboard also now need to be tracked

Not really. They are optional dependencies. We really need pyth to be up since it is what we use in the dip program
